Documentation/colors/literals-swift3.md

```diff
@@ -0,0 +1,45 @@
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File name | colors/literals-swift3.stencil |
+| Invocation example | `swiftgen colors -t literals-swift3 …` |
+| Language | Swift 3 |
+| Author | Olivier Halligon |
+
+## When to use it
+
+- When you need to generate *Swift 3* code
+- Supports _multiple_ color names with the _same_ value
+- Uses `#colorLiteral`s for easy preview of the actual color
+
+## Customization
+
+You can customize some elements of this template by overriding the following parameters when invoking `swiftgen` in the command line, using `--param <paramName>=<newValue>`
+
+| Parameter Name | Default Value | Description |
+| -------------- | ------------- | ----------- |
+| `enumName` | `ColorName` | Allows you to change the name of the generated `enum` containing all colors. |
+
+## Generated Code
+
+**Extract:**
+
+```swift
+extension ColorName {
+  /// 0x339666ff (r: 20%, g: 58%, b: 40%, a: 100%)
```

Comment:
I still think that a lot of people dealing with colors palettes provided by their designers (e.g. via a Sketch document or a sheet listing all colors of the app theme, etc) generally deal either with hex or with `rgba(51, 150, 104, 255)` and less commonly with % for rgb values :wink: